src/Kunstmaan/MediaBundle/Validator/Constraints/HasGuessableExtensionValidator.php

```diff
@@ -60,6 +66,7 @@ public function validate($value, Constraint $constraint)
      */
     public function setExtensionGuesser(ExtensionGuesserFactoryInterface $extensionGuesserFactory)
     {
+        @trigger_error('Calling the setExtensionGuesser method of this service is deprecated since KunstmaanMediaBundle 5.6 and will be replaced by the session in KunstmaanMediaBundle 6.0.', E_USER_DEPRECATED);
```

Comment:
```suggestion
        @trigger_error('Calling the setExtensionGuesser method of this service is deprecated since KunstmaanMediaBundle 5.6 and will be replaced by the "@mime.types" service in KunstmaanMediaBundle 6.0.', E_USER_DEPRECATED);
```

same for the setter bellow